tests/automated_install.sh

```diff
@@ -65,53 +70,83 @@ fi
 create_cluster_VMs || ( echo "############## VBOX CREATE CLUSTER VMs RETURNED $? ##############" && exit 1 )
 install_cluster || ( echo "############## VBOX CREATE INSTALL CLUSTER RETURNED $? ##############" && exit 1 )
 
+printf "#### Install ruby gems\n"
+vagrant ssh -c 'cd chef-bcpc; PATH=/opt/chefdk/embedded/bin:/usr/bin:/bin bundle install --path vendor/bundle'
+
 printf "#### Cobbler Boot\n"
 printf "Snapshotting pre-Cobbler and booting (unless already running)\n"
+
+if [ "$CLUSTER_TYPE" == "Kafka" ]; then
+  VM_LIST=(bcpc-vm1 bcpc-vm2 bcpc-vm3 bcpc-vm4 bcpc-vm5 bcpc-vm6)
```

Comment:
In prod, we don't run ZK and Kafka on the same nodes.  6 nodes gives us 3 for ZK and 3 for Kafka.